assets/kube-apiserver/kube-apiserver-secret.yaml

```diff
@@ -0,0 +1,18 @@
+apiVersion: v1
```

Comment:
Does this file need to be added to the list of files rendered [here](https://github.com/openshift/ibm-roks-toolkit/blob/master/pkg/render/manifests.go#L88)?

Reply:
Secrets are handled in openshift-master's [cluster-secrets](https://github.ibm.com/alchemy-containers/armada-openshift-master/tree/master/roles/cluster-secrets) role, so they don't need to be included in this change.